src/meta/filetype.rs

```diff
@@ -42,6 +45,26 @@ impl FileType {
             FileType::Special
         }
     }
+
+    #[cfg(windows)]
+    pub fn new(meta: &Metadata, permissions: &Permissions) -> Self {
+        let file_type = meta.file_type();
+
+        if file_type.is_file() {
+            FileType::File {
+                exec: permissions.is_executable(),
+                uid: permissions.setuid,
+            }
+        } else if file_type.is_dir() {
```

Comment:
I would recommend avoiding if\else statements, especially if\elseif. 
In case there would be a new file type, you wouldn't catch it via a compiler, but you could catch it if you would use pattern matching. And you already used 'better' approach here https://github.com/Peltoche/lsd/blob/master/src/meta/mod.rs#L50

Reply:
Hi @humb1t , thanks for the comment.

I don't think it's possible because it's a `fs::Metadata::FileType` and not a `lsd::meta::filetype::Filetype` enum like your example.